mteb/models/nomic_models.py

```diff
@@ -0,0 +1,22 @@
+from functools import partial
+
+from mteb.model_meta import ModelMeta, sentence_transformers_loader
+
+nomic_embed = ModelMeta(
+    loader=partial(
+        sentence_transformers_loader,
+        trust_remote_code=True,
+        model_name="nomic-ai/nomic-embed-text-v1.5",
+        revision=None,
+    ),
```

Comment:
Do their models not require some extra normalization?

See https://huggingface.co/nomic-ai/nomic-embed-text-v1.5#sentence-transformers

Reply:
I have added this in the latest PR